detectors/metricshub/README.md

```diff
@@ -0,0 +1,18 @@
+# MetricsHub Detectors
+
+This folder contains detectors that may be used to trigger events based on the metrics collected by MetricsHub.
+
+Please note that you may want or need different thresholds than those provided here.
+You may also want to create your own detectors.
+
+## Importing Detectors
+Two options exist:
+1. Edit and send the Detector JSON [via API](https://dev.splunk.com/observability/reference/api/detectors/latest#endpoint-create-single-detector)
+  ```
+  curl -X POST "https://api.{REALM}.signalfx.com/v2/detector" \
+    -H "Content-Type: application/json" \
+    -H "X-SF-TOKEN: <value>" \
+    -d @"/path/to/detector/detector_name.json"
+  ```
+2. Copy the detector's JSON and paste it into your own Detector [via the UI](https://docs.splunk.com/Observability/alerts-detectors-notifications/create-detectors-for-alerts.html#nav-Create-detectors-to-trigger-alerts)
+
```

Comment:
Sorry Marc one last thing. Could you please add the disclaimer about support to this README as well? A link to your support or contact page may also be helpful in those disclaimers?

Thanks again for your contribution!

- [ ] Add Disclaimer for detectors

Reply:
@greatestusername-splunk , I added a disclaimer, support information and a link to the MetricsHub web site.
I hope this is okay with you.

Thank you!